doc/source/control.rst

```diff
@@ -38,28 +38,27 @@ Programming is hard when we have to juggle many code elements of each type at
 the same time.  Good programming is about managing these three elements so that
 the developer is only required to think about a handful of them at a time.  For
 example we might collect many integer variables into a list of integers or
-build a big function out of smaller ones.  While we have natural ways to manage
-data and functions, control flow presents more of a challenge.
+build a big function out of smaller ones.
 
 We organize our data into **data structures** like lists, dictionaries, or objects
 in order to group related data together -- this allows us to manipulate large
 collections of related data as if we were only manipulating a single entity.
 
-We **build large functions out of smaller ones**; enabling us to break up a
+We **build large functions out of smaller ones**, enabling us to break up a
 complex task like doing laundry into a sequence of simpler tasks.
 
 .. code::
 
     def do_laundry(clothes):
-        wet_clothes = wash(clothes, coins)
-        dry_clothes = dry(wet_clothes, coins)
+        wet_clothes = wash(clothes)
+        dry_clothes = dry(wet_clothes)
         return fold(dry_clothes)
```

Comment:
I feel the presence of `coins` is somewhat antithetical to the principal of _function purity_ as described [in the docs](https://toolz.readthedocs.io/en/latest/purity.html#examples).